docs/source/getting_started/lightly_at_a_glance.rst

```diff
@@ -64,11 +64,20 @@ with the normalized temperature-scaled cross entropy loss and simple stochastic
 
 .. code-block:: python
 
+    import torchvision
+
     import lightly.models as models
     import lightly.loss as loss
 
-    # build a resnet-34 with 32 embedding neurons
-    model = models.ResNetSimCLR(name='resnet-34', num_ftrs=32)
+    # use a resnet backbone
+    resnet = torchvision.models.resnet.resnet18()
+    resnet = nn.Sequential(
+        *list(resnet.children())[:-1],
+        nn.AdaptiveAvgPool2d(1),
```

Comment:
I think this is redundant - the last 2 layers of ResNet18 are:

```
AdaptiveAvgPool2d-67            [-1, 512, 1, 1]               
Linear-68                       [-1, 1000]
```

So this would just be stripping the linear layer and adding another Adaptive Avg Pool?

Reply:
You are right, the Adaptive Avg Pool is redundant. Thanks!